docs/notifications/01-build/10-Develop-Fetch-Notifications.mdx

```diff
@@ -103,3 +104,125 @@ const inboxNotifications = await userAlice.notification.list("INBOX");
 ```
```

Comment:
- Settings are not shown properly, shows Boolean Setting
- Should be under UI Components just like in chat